src/gmp-test/ChainBTokenSwapper.sol

```diff
@@ -4,31 +4,31 @@ pragma solidity 0.8.9;
 
 import { IERC20 } from '../interfaces/IERC20.sol';
 
-contract TokenSwapper {
+contract ChainBTokenSwapper {
```

Comment:
Why can't there be one token swapper on each chain that handles the functionality needed on the source side and destination side?

Reply:
Token swapper here is the analogy of preexisting Uniswap. Renamed it to `DestinationChainTokenSwapper`